notebook-lessons/oss-module-01-git.ipynb

```diff
@@ -2709,6 +2712,246 @@
     "\n",
     "The vast majority of the time, you'll be doing a `pull` instead of separate `fetch` and `merge`.  But sometimes -- like if your local history is messy -- it can be helpful to do a fetch and see what's happening before you merge. "
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4dd7105f",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "# 6: Git Commit Signing and DCO Signoff\n",
+    "\n",
+    "<img src=\"../img/icon-gears.png\" width=\"200\" />\n",
+    "\n",
+    "In professional and enterprise environments, you may encounter requirements for **signed commits** and **DCO signoff**. These are important security and legal compliance measures that help establish trust and legal clarity in collaborative projects."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c62e3d6b",
+   "metadata": {},
+   "source": [
+    "## 6.1: Developer Certificate of Origin (DCO) Signoff\n",
+    "\n",
+    "### What is `git commit --signoff`?\n",
+    "\n",
+    "The `--signoff` flag adds a \"Signed-off-by\" line to your commit message, which is a **requirement in some projects** - most notably the Linux kernel and official IBM projects.\n",
+    "\n",
+    "This practice originated in response to the [SCO lawsuit(s) versus IBM](https://en.wikipedia.org/wiki/SCO%E2%80%93Linux_disputes) and helps to solve concerns around copyright infringement. The sign-off effectively serves as a **Developer Certificate of Origin (DCO)**, asserting that:\n",
```

Comment:
I suggest this instead.  There were several stakeholders involved in the series of lawsuits and countersuits

> In the Linux kernel, this practice originated from lawsuits and countersuits between [SCO lawsuit(s) and Linux vendors](https://en.wikipedia.org/wiki/SCO%E2%80%93Linux_disputes) and helps to resolve concerns around copyright infringement. The sign-off effectively serves as a **Developer Certificate of Origin (DCO)**, asserting that